src/components/composer/index.js

```diff
@@ -597,49 +591,15 @@ class ComposerWithData extends Component<Props, State> {
       !networkOnline ||
       (websocketConnection !== 'connected' &&
         websocketConnection !== 'reconnected');
-
+    const showSelectedCommunity = activeCommunity
+      ? activeCommunity.length > 0
+      : false;
+    const showSelectedChannel = activeChannel
```

Comment:
`showSelectedChannel` and `showSelectedCommunity` are unused atm. Dont know if you plan on using them? :)

Reply:
No, I need to remove them, thanks @littleStudent .